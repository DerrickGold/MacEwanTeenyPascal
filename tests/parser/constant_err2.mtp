(* constants *)
const test :=10;
const floop := 1337;
(*three := 'Stringify';*)

begin

end
.