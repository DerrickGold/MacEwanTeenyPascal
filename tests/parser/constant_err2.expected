Syntax Error: line 3:
Unexpected token (TOK_KEY_CONST, 'const'), likely missing a(n)  
  'TOK_KEY_BEGIN' or
  'TOK_ID' or
  'TOK_KEY_VAR' or
Possibly due to malformed or missing:
  Non-terminal: 'If Statement' or
  Non-terminal: 'While Statement' or
  Non-terminal: 'Write Statement' or
  Non-terminal: 'Read Statement' or
  Non-terminal: 'Case'
